core/hash/fetch_values_spec.rb

```diff
@@ -0,0 +1,33 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/classes', __FILE__)
+
```

Comment:
You should guard here that this is a 2.3 feature with
`ruby_version_is "2.3" do`
(and indent accordingly + `end`)